packages/orchestrator-ui-components/src/components/WFOProcessesList/processListObjectMappers.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    GraphqlFilter,
+    GraphQLSort,
+    Process,
+    ProcessesResult,
+} from '../../types';
+import { ProcessListItem } from './WFOProcessList';
+
+export const mapGraphQlProcessListResultToProcessListItems = (
+    processesResult: ProcessesResult,
+): ProcessListItem[] =>
+    processesResult.processes.page.map((process) => {
+        const {
+            workflowName,
+            lastStep,
+            lastStatus,
+            workflowTarget,
+            createdBy,
+            assignee,
+            processId,
+            startedAt,
+            lastModifiedAt,
+            subscriptions,
+            product,
+            customer,
+        } = process;
+
+        return {
+            workflowName,
+            lastStep,
+            lastStatus,
+            workflowTarget,
+            createdBy,
+            assignee,
+            processId,
+            startedAt,
+            lastModifiedAt,
+            subscriptions,
+            productName: product?.name,
+            productTag: product?.tag,
+            customer: customer.fullname,
+            customerAbbreviation: customer.shortcode,
+        };
+    });
+
+// Some fields are not a key of Process, however backend still supports them
+// Backend concatenates object name with the key, e.g. product.name becomes productName
+// Todo: typecast is needed until ticket is implemented:
+// https://github.com/workfloworchestrator/orchestrator-ui/issues/290
+const fieldMapper = (field: keyof ProcessListItem): keyof Process => {
+    switch (field) {
+        case 'customer':
+            return 'customerFullname' as keyof Process;
```

Comment:
Is it an option to return customer.fullName here? That makes it a little clearer to me. Or wont that work?